client/cpp/kserve-api/README.md

```diff
@@ -0,0 +1,159 @@
+```
+
+### Build client library and samples
+```Bash
+cmake . && make
+```
+
+## GRPC Examples <a name="grpc-api"></a>
+
+
+## GRPC Examples with Resnet Model
+
+### Download the Pretrained Model
+Download the model files and store them in the `models` directory
+```Bash
+mkdir -p models/resnet/1
+curl https://storage.openvinotoolkit.org/repositories/open_model_zoo/2022.1/models_bin/2/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.bin https://storage.openvinotoolkit.org/repositories/open_model_zoo/2022.1/models_bin/2/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.xml -o models/resnet/1/resnet50-binary-0001.bin -o models/resnet/1/resnet50-binary-0001.xml
+```
+
+### Start the Model Server Container with Resnet Model
+```Bash
+docker run --rm -d -v $(pwd)/models:/models -p 9000:9000 openvino/model_server:latest --model_name resnet --model_path /models/resnet --port 9000 
+```
+
+Once you finish above steps, you are ready to run the samples.
+
+### Run the Client to get server liveness <a name="grpc-server-live"></a>
+
+- Command
+
+```Bash
+./grpc_server_live --help
+Sends requests via KServe gRPC API to check if server is alive.
+Usage:
+  grpc_infer_dummy [OPTION...]
+
+  -h, --help              Show this help message and exit
+      --grpc_address arg  Specify url to grpc service.  (default: 
+                          localhost)
+      --grpc_port arg     Specify port to grpc service.  (default: 9000)
+      --timeout arg       Request timeout. (default: 0)
+```
+
+- Usage Example 
+
+```Bash
+./grpc_server_live --grpc_port 9000 --grpc_address localhost
+Server Live: True
+```
+
+### Run the Client to perform inference
+```Bash
+./grpc_infer_resnet --help
+Sends requests via KServe gRPC API.
+Usage:
+  grpc_infer_resnet [OPTION...]
+
+  -h, --help                    Show this help message and exit
+      --images_list IMAGES      Path to a file with a list of labeled 
+                                images. 
+      --labels_list LABELS      Path to a file with a list of labels. 
+      --grpc_address GRPC_ADDRESS
+                                Specify url to grpc service.  (default: 
+                                localhost)
+      --grpc_port PORT          Specify port to grpc service.  (default: 
+                                9000)
+      --input_name INPUT_NAME   Specify input tensor name.  (default: 0)
+      --output_name OUTPUT_NAME
+                                Specify input tensor name.  (default: 1463)
+      --model_name MODEL_NAME   Define model name, must be same as is in 
+                                service.  (default: resnet)
+      --model_version MODEL_VERSION
+                                Define model version.
+      --timeout TIMEOUT         Request timeout. (default: 0)
+```
+
+- Usage Example
+
+```Bash
+./grpc_infer_resnet --images_list resnet_input_images.txt --labels_list resnet_labels.txt --grpc_port 9000  
+../../../../demos/common/static/images/airliner.jpeg classified as 404 airliner 
```

Comment:
can we display only the filename? full path with so many dots is not easy to read

Reply:
to only have filenames we would need to run the script from the catalog with images which will make the command longer and add additional step
manually modifying the output will prevent documentation tests from working properly